R/distance.R

```diff
@@ -2,13 +2,15 @@
 #'
 #' @importFrom stringdist stringdistmatrix
 #' @importFrom purrr map pmap map2
-#' @importFrom magrittr %>%
+#' @importFrom magrittr %>% %<>% set_attr
+#' @importFrom tidyr unite
+#' @importFrom dplyr select_if group_keys group_map group_by group_by_at
 
 #' @description Computing sequential distances between clonotypes from two repertoires:
 #'
 #' @usage
 #'
-#' seqDist(.data, .col = 'CDR3.nt', .method = 'hamming', ...)
+#' seqDist(.data, .col = 'CDR3.nt', .method = 'hamming', .group_by = c("V.name", "J.name"), .group_by_seqLength = TRUE  ...)
```

Comment:
CRAN check shows error, probably because of mismatch between function documentation and function itself:
```
* checking Rd \usage sections ... WARNING
Bad \usage lines found in documentation object 'seqDist':
  seqDist(.data, .col = 'CDR3.nt', .method = 'hamming', .group_by = c("V.name", "J.name"), .group_by_seqLength = TRUE  ...)

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

You can test the code and docs for CRAN with this script:
```
cd /path/to/immunarch
R -e "devtools::document(roclets = c('"'"'rd'"'"', '"'"'collate'"'"', '"'"'namespace'"'"')); devtools::build()"
cd ..
R CMD check --as-cran $(ls -1 immunarch_*)
```

Reply:
Maybe, it's just a missing `,` before `...`